[2022-12-25T19:50:58.116+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-10-17T15:00:00+00:00 [queued]>
[2022-12-25T19:50:58.135+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-10-17T15:00:00+00:00 [queued]>
[2022-12-25T19:50:58.136+0300] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:50:58.136+0300] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2022-12-25T19:50:58.137+0300] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:50:58.156+0300] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): first_task> on 2022-10-17 15:00:00+00:00
[2022-12-25T19:50:58.160+0300] {standard_task_runner.py:55} INFO - Started process 33705 to run task
[2022-12-25T19:50:58.170+0300] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_pivot', 'first_task', 'scheduled__2022-10-17T15:00:00+00:00', '--job-id', '710', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_total.py', '--cfg-path', '/var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T/tmplro2g04g']
[2022-12-25T19:50:58.173+0300] {standard_task_runner.py:83} INFO - Job 710: Subtask first_task
[2022-12-25T19:50:58.199+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:50:58.201+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_alchemy_conn] has been deprecated, you should use[database/sql_alchemy_conn] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:50:58.203+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:400 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:50:58.206+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_engine_encoding] has been deprecated, you should use[database/sql_engine_encoding] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:50:58.241+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/utils/sqlalchemy.py:121 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:50:58.291+0300] {task_command.py:389} INFO - Running <TaskInstance: titanic_pivot.first_task scheduled__2022-10-17T15:00:00+00:00 [running]> on host mbp-vadim.bbrouter
[2022-12-25T19:50:58.396+0300] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-10-17T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-17T15:00:00+00:00
[2022-12-25T19:50:58.399+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T
[2022-12-25T19:50:58.400+0300] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "Here we start!"']
[2022-12-25T19:50:58.420+0300] {subprocess.py:86} INFO - Output:
[2022-12-25T19:50:58.421+0300] {subprocess.py:93} INFO - Here we start!
[2022-12-25T19:50:58.422+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-25T19:50:58.529+0300] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=first_task, execution_date=20221017T150000, start_date=20221225T165058, end_date=20221225T165058
[2022-12-25T19:50:58.596+0300] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-25T19:50:58.666+0300] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
